app/routers/systems.py

```diff
@@ -64,8 +64,44 @@ async def get_systems(
 
 
 @router.post("", response_model=SystemRead, status_code=status.HTTP_201_CREATED)
-async def create_system(data: SystemCreate):  # pragma: no cover
-    pass
+async def create_system(
+    data: SystemCreate,
+    account_repo: AccountRepository,
+    system_repo: SystemRepository,
+    auth_ctx: CurrentAuthContext,
+):
+    if auth_ctx.account.type == AccountType.AFFILIATE and data.owner.id != auth_ctx.account.id:
```

Comment:
Like we have for Entitlements, data.owner is mandatory only for payloads sent by an Operations account and must not be included in payloads sent by an Affiliate account. 
Could you please implement it this way ?

Reply:
The JWT token should be returned only as the response of the create endpoint and of the update in the case it is received in the request. In all the other endpoints it should be omitted. Could you please check that ?